src/components/Exercise/List.js

```diff
@@ -25,15 +35,26 @@ export default function List({ exercises, handleExerciseSelect }) {
     }
     return acc
   }, [])
-
   return (
     <>
       <S.ButtonWrapper>
-        <Button as={Link} to={`${url}/add`}>
+        <Button primary="true" as={Link} to={`${url}/add`}>
           Add
         </Button>
+        <SearchBox
+          register={register}
+          onSubmit={handleSubmit(onSubmit)}
+          reset={reset}
+          search={search}
+        />
       </S.ButtonWrapper>
-      <S.Wrapper>{renderExercises}</S.Wrapper>
+      <S.Wrapper>
+        {exercises.length !== 0 ? (
```

Comment:
`exercise.length !== 0` is the same as asking if `exercise.length` is truthy

Reply:
thanks, I totally overlooked his, changed it immediatly